pkg/k8sutil/services.go

```diff
@@ -24,6 +24,11 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+const (
+	ServiceIpAddressOverrideAnnotation = "service.banzaicloud.io/ip-address-override"
+	ServiceHostnameOverrideAnnotation  = "service.banzaicloud.io/hostname-override"
```

Comment:
If they do not need to be exported to other files, then:
```suggestion
	serviceIpAddressOverrideAnnotation = "service.banzaicloud.io/ip-address-override"
	serviceHostnameOverrideAnnotation  = "service.banzaicloud.io/hostname-override"
```